lexicons/app/bsky/administration/moderationAction.json

```diff
@@ -0,0 +1,21 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.administration.moderationAction",
+  "defs": {
+    "view": {
+      "type": "object",
+      "required": ["id", "action", "subject", "rationale", "createdBy", "createdAt"],
+      "properties": {
+        "id": {"type": "integer"},
+        "action": {"type": "string", "knownValues": ["takedown"]},
+        "subject": {"type": "union", "refs": ["app.bsky.actor.ref"]},
+        "rationale": {"type": "string"},
```

Comment:
Boy I'm going to bikeshed all over this. Why rationale vs reason?

Reply:
It was just the word in my head for this, all good changing it.